napari/components/viewer_model.py

```diff
@@ -115,6 +116,7 @@ class ViewerModel(KeymapProvider, MousemapProvider, EventedModel):
     text_overlay: TextOverlay = Field(
         default_factory=TextOverlay, allow_mutation=False
     )
+    overlays: Overlays = Field(default_factory=Overlays, allow_mutation=False)
```

Comment:
I like this, in the future we can refactor to include more overlays in here. Thoughts on that @jni? Shouldn't come in this PR